tests/fireproof/hello.test.ts

```diff
@@ -23,14 +23,14 @@ describe("hello public API", () => {
   });
   beforeEach(async () => {
     await sthis.start();
-    db = fireproof("test-public-api");
+    db = fireproofDB("test-public-api");
```

Comment:
this all looks great, except I'm confused about the introduction of this entrypoint